superset-frontend/src/SqlLab/reducers/getInitialState.js

```diff
@@ -183,9 +186,15 @@ export default function getInitialState({
           },
         };
       });
-      sqlLab.tables.forEach(table =>
-        tables.push({ ...table, inLocalStorage: true }),
-      );
+      sqlLab.tables.forEach(table => {
```

Comment:
Nit, could this be changed to use `reduce` so it's clear what the intention is while iterating over the array and manipulating the new type?

Reply:
good call. I will update this part